studio/src/rte/rte-field.js

```diff
@@ -0,0 +1,746 @@
+import { LitElement, html, nothing, css } from 'lit';
```

Comment:
shouldn't this file go with ./fields folder?

Reply:
I didn't want to have fields/rte.
I find it easier to access from src/rte.
can we discuss it later? :)